src/script/components/asset/controls/mediaButton.js

```diff
@@ -32,37 +33,28 @@ class MediaButtonComponent {
    * @param {Object} component_info - Component information
    */
   constructor(params, component_info) {
-    this.dispose = this.dispose.bind(this);
     this.media_element = params.src;
     this.large = params.large;
     this.asset = params.asset;
     this.uploadProgress = params.uploadProgress;
     this.transferState = params.transferState;
 
+    this.dispose = this.dispose.bind(this);
+    this.on_play = this.on_play.bind(this);
+    this.on_pause = this.on_pause.bind(this);
+
     if (this.large) {
       component_info.element.classList.add('media-button-lg');
     }
 
     this.media_is_playing = ko.observable(false);
 
-    this.on_play_button_clicked = function() {
-      if (typeof params.play === 'function') {
-        params.play();
-      }
-    };
-    this.on_pause_button_clicked = function() {
-      if (typeof params.pause === 'function') {
-        params.pause();
-      }
-    };
-    this.on_cancel_button_clicked = function() {
-      if (typeof params.cancel === 'function') {
-        params.cancel();
-      }
-    };
+    const noop = () => {};
+
+    this.on_play_button_clicked = typeof params.play === 'function' ? () => params.play() : noop;
+    this.on_pause_button_clicked = typeof params.pause === 'function' ? () => params.pause() : noop;
+    this.on_cancel_button_clicked = typeof params.cancel === 'function' ? () => params.cancel() : noop;
```

Comment:
what about making that cameCase while we at it?

Reply:
yep, sounds like a good idea ;)